src/components/20-molecules/top-content-bar/story.js

```diff
@@ -48,22 +48,11 @@ export const TopContentBar = () => {
     'Unidentified flying object detected in your region. People are panicking. Stay calm!'
   );
   const link = text('Add axa-link', '');
-  const barClosed = !!window.sessionStorage.getItem('top-content-bar-closed');
-
-  setTimeout(() => {
-    const el = document.querySelector('axa-top-content-bar');
-    if (!el) {
-      return;
-    }
-
-    el.addEventListener('axa-top-content-bar-close', () => {
-      window.sessionStorage.setItem('top-content-bar-closed', 'true');
-    });
-  });
 
   const template = html`
     <h1>HEADER</h1>
     <axa-top-content-bar
+      style="display:none"
```

Comment:
If above advice on FOUC treatment is heeded, this line can be deleted.

Reply:
Deleted. @markus-walther With all these last changes I just realised that in this PR I just deleted the code that we don't need. I actually did not add any new functionality. Everything is moved to AEM code.